<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1324</width>
    <height>978</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QWidget" name="widget_10" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Projekt zaliczeniowy z przedmiotu</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Algorytmy grafowe</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_11" native="true">
      <property name="minimumSize">
       <size>
        <width>1280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1280</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item alignment="Qt::AlignLeft|Qt::AlignTop">
        <widget class="QToolBox" name="toolBox">
         <property name="minimumSize">
          <size>
           <width>300</width>
           <height>360</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>360</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="page_1">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>292</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Budowanie grafu</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QWidget" name="okno_add_del" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QRadioButton" name="rb_dodaj">
                <property name="text">
                 <string>Dodaj</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
                <property name="autoExclusive">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_usun">
                <property name="text">
                 <string>Usuń</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="okno_V_E" native="true">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QRadioButton" name="rb_V">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>Wierzchołek</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_E">
                <property name="text">
                 <string>Krawędź</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="okno_skie_nieskie" native="true">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QRadioButton" name="rb_skie">
                <property name="text">
                 <string>Skierowana</string>
                </property>
                <property name="checkable">
                 <bool>true</bool>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="rb_nieskie">
                <property name="text">
                 <string>Nieskierowana</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_4" native="true">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="b_edytuj">
                <property name="text">
                 <string>Edytuj</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="b_koniec">
                <property name="text">
                 <string>Koniec budowy</string>
                </property>
               </widget>
              </item>
             </layout>
             <zorder>b_koniec</zorder>
             <zorder>b_edytuj</zorder>
            </widget>
           </item>
           <item>
            <widget class="QWidget" name="widget_5" native="true">
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QPushButton" name="b_clean">
                <property name="text">
                 <string>Wyczyść graf</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="page_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>300</width>
            <height>292</height>
           </rect>
          </property>
          <attribute name="label">
           <string>Obliczenia</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>107</height>
              </size>
             </property>
             <property name="text">
              <string>Wybór algorytmu</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="BFS_but">
             <property name="text">
              <string>BSF</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="DFS_but">
             <property name="text">
              <string>DFS</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_3">
             <property name="text">
              <string>????????</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox"/>
           </item>
           <item>
            <widget class="QTextEdit" name="wynik">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOn</enum>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>2</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>2</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MyWidget" name="ekran" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>950</width>
           <height>400</height>
          </size>
         </property>
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget_9" native="true">
      <property name="minimumSize">
       <size>
        <width>1300</width>
        <height>300</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>304</width>
        <height>700</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QWidget" name="okno_LS" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Lista sąsiadów</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="list_LS">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="okno_MI" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Macierz incydencji</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="MI">
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="okno_MP" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Macierz przyległości </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="MP">
            <property name="font">
             <font>
              <family>Arial</family>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOn</enum>
            </property>
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
      <zorder>okno_MI</zorder>
      <zorder>okno_LS</zorder>
      <zorder>okno_MP</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1324</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyWidget</class>
   <extends>QWidget</extends>
   <header location="global">mywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
